site/assets/ocrd.css

```diff
@@ -41,6 +41,10 @@ h2 code, h3 code, h4 code {
     text-align: center;
 }
 
+ul.no-bullets {
+	list-style-type: none;
+}
```

Comment:
```suggestion
article.tile ul {
  list-style-type: none !important
}
```

![image](https://user-images.githubusercontent.com/273367/117330300-d3b2c280-ae95-11eb-961d-af0a5170a8ff.png)

or just add `!important` to your CSS rule, otherwise it looks like this

![image](https://user-images.githubusercontent.com/273367/117330526-1379aa00-ae96-11eb-9e0c-42cb369a5dac.png)